CHANGELOG.md

```diff
@@ -4,14 +4,96 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## Version 4.0.0
+
+The latest stable release of ink! is here 🥳
+
+This version brings a lot of usability improvements, making the language better suited
+for the needs of production parachains.
+
+A couple of highlights include:
+- Changes to how contract storage works, which significantly reduced the sizes of
+  contract binaries
+- A new end-to-end testing framework, letting you easily write integration tests
+- Changes to the metadata format, which (in part) makes error handling more expressive
+
+There's a lot more to dig through, so take some time to poke around the `CHANGELOG`
+(including the `4.0.0-alpha` and `4.0.0-beta` releases).
+
+You may notice there have been a few breaking changes. No need to be scared though, we
+wrote up a [migration guide](https://use.ink/faq/migrating-from-ink-3-to-4) covering all
+of the breaking changes and how to update your ink! 3.x contract accordingly.
+
+Thanks to everyone that helped make this release possible ❤️
+
+### Compatibility
+
+In order to build contracts which use ink! `v4.0.0` you need to use
+`cargo-contract`
+[`v2.0.0`](https://github.com/paritytech/cargo-contract/releases/tag/v2.0.0).
+You can install it as follows:
+
+`cargo install cargo-contract --forced --locked`
+
+You will also need to use a version of [`pallet-contracts`](https://github.com/paritytech/substrate/tree/master/frame/contracts)
+later than [TODO](https://github.com/paritytech/substrate/commit/todo)
```

Comment:
@ascjones can you fill these in? I didn't have time to check what these were

Reply:
https://use.ink/faq/migrating-from-ink-3-to-4#compatibility

The compatible versions are noted here.

Reply:
Thanks have added in the link to that and updated the requirement versions